native/cocos/core/assets/TextureCube.h

```diff
@@ -203,7 +203,7 @@ class TextureCube final : public SimpleTexture {
         return _mipmaps.empty() ? nullptr : &_mipmaps[0];
     }
 
-    void setImage(const ITextureCubeMipmap &value);
+    void setImage(const ITextureCubeMipmap *value);
```

Comment:
Getter `TextureCube::getImage` return `const ITextureCubeMipmap*`, setter needs to pass in pointer too.
It's the same logic as which in web and make swig binding work.